apps/aehttp/priv/swagger.json

```diff
@@ -597,12 +598,13 @@
           "required" : true,
           "type" : "string"
         }, {
-          "name" : "tx_objects",
+          "name" : "tx_encoding",
           "in" : "query",
           "description" : "Transactions encoding: MessagePack (default) or JSON",
           "required" : false,
-          "type" : "boolean",
-          "default" : false
+          "type" : "string",
+          "default" : "message_pack",
+          "enum" : [ "message_pack", "json" ]
```

Comment:
Maybe a Swagger definition may have avoided repetition? Though totally optional of course.

Reply:
I've just tried this but for some reason current swagger does not accept query params to be schema refs. For example
```                                                                          
  /block-by-hash:                                                                                    
    get:
      tags:
        - external 
      operationId: GetBlockByHash 
      description: 'Get a block by hash'                                                             
      consumes:
        - application/json                                                                           
      produces:
        - application/json                                                                           
      parameters: 
        - in: query
          name: hash
          description: 'Hash of the block to fetch'                                                  
          required: true                                                                             
          schema: 
            $ref: '#/definitions/EncodedHash'            
````
is invalid, no matter that  `'#/definitions/EncodedHash' ` is a valid definition